src/utils.py

```diff
@@ -0,0 +1,56 @@
+from typing import Union
+from io import BytesIO
+from PIL import Image
+
+import numpy as np
+
+from viam.logging import getLogger
+from viam.media.video import CameraMimeType
+from viam.media.video import ViamImage
```

Comment:
Can't these two lines just be `from viam.media.video import CameraMimeType, ViamImage` like you have in the tests?